web/replacement_terms.json

```diff
@@ -0,0 +1,71 @@
+{
```

Comment:
Thinking about the scoring step, I think you actually want to invert this file, so it's something like this:

```
{
  "abs": ["positive", "magnitude"],
  "acos": ["arccos"],
  ...
  "polygon": [
    {
      "value": "triangle",
      "explanation": "A triangle is a polygon with three vertices."
    },
    "quadrilateral",
    "parallelogram",
    "trapezoid",
    "rhombus",
    "pentagon",
    "hexagon",
    "heptagon",
    "octagon",
  ],
  ...
}
```